std/range/package.d

```diff
@@ -3231,18 +3225,42 @@ private:
     else
         alias fun = Fun[0];
 
+    import std.traits: ReturnType, functionAttributes, FunctionAttribute;
```

Comment:
haha, when you get around to moving imports to locals for this module, this will help, though 😄